apps/ledger-live-mobile/src/actions/settings.ts

```diff
@@ -199,10 +200,14 @@ export const setLastSeenCustomImage = ({
 export const clearLastSeenCustomImage = () =>
   setLastSeenCustomImageAction({ imageSize: 0, imageHash: "" });
 
-const completeOnboardingAction = createAction(
-  SettingsActionTypes.SETTINGS_COMPLETE_ONBOARDING,
-);
-export const completeOnboarding = () => completeOnboardingAction();
+const completeOnboardingAction =
+  createAction<SettingsCompleteOnboardingPayload>(
+    SettingsActionTypes.SETTINGS_COMPLETE_ONBOARDING,
+  );
+export const completeOnboarding = (hasCompletedOnboarding = true) =>
+  completeOnboardingAction({
+    hasCompletedOnboarding,
+  });
```

Comment:
added a way to reset hasCompletedOnboarding to false, for debugging purposes (see changes in the associated reducer and in the debug settings with the `ResetOnboardingStateRow` component